current/runtime/fees.md

```diff
@@ -4,118 +4,151 @@ lang: en
 title: Transaction Fees
 ---
 
-When transactions are submitted to a blockchain, they are executed by the nodes in the network. To
-be economically sustainable, nodes charge a fee to execute a transaction. This fee must be covered
-by the sender of the transaction. The cost to execute transactions can vary, so Substrate provides a
-flexible mechanism to characterize the minimum cost to include a transaction in a block.
+When transactions (extrinsics) are submitted to a blockchain, they are executed (dispatched) by the
+nodes in the network. To be economically sustainable, nodes charge a fee to dispatch a transaction.
+This fee must be covered by the sender of the transaction. The cost to dispatch transactions can
+vary, so Substrate provides a flexible mechanism called "weights" to characterize the minimum cost to
+include a transaction in a block.
```

Comment:
This paragraph is a bit off.
- "transactions" and "extrinsics" are not interchangeable. Transactions are one class of extrinsic.
- "execute" and "dispatch" are also not interchangeable. "Dispatch" means to invoke, while execute means to actually carry out the requested function call.
- "The cost to dispatch transactions can vary, so Substrate provides a flexible mechanism called "weights" to characterize the minimum cost to include a transaction in a block." You've put the cart before the horse in two ways here.
    - "The cost to dispatch transactions can vary" is not true as an antecedent. "The time to _execute_ transactions can vary" is an antecedent, and its consequent is that we charge variable transaction fees. 
    - ""weights" to characterize the minimum cost" is not true. Weight characterizes the time to execute a transaction and is used as one component in charging a fee to include it.